server/filestore.go

```diff
@@ -107,6 +107,35 @@ func (alg StoreCompression) String() string {
 	}
 }
 
+func (alg StoreCompression) MarshalJSON() ([]byte, error) {
+	var str string
+	switch alg {
+	case S2Compression:
+		str = "s2"
```

Comment:
I suggest S2, I think that is how they refer to it.

Reply:
I stuck with the existing pattern from the JS schema here, which is that the values are all lower-case, although there is a separate `String()` function in the `jsm.go` PR that is for pretty-printing in the CLI (`"S2 compression"`) so a CLI user never sees it.